src/lib/components/Filteration.svelte

```
@@ -0,0 +1,134 @@
+	import { goto } from '$app/navigation';
+
+	type Filter = { id: string; name: string };
+
+	let { items = $bindable(), itemList = $bindable(), itemName, label = null, ...rest } = $props();
+	let open = $state(false);
+	let searchQuery = $state('');
+	const placeholder = 'Select ' + (label ?? itemName) + 's';
+
+	// Debounced search
+	let searchTimeout: NodeJS.Timeout | undefined;
+	function handleSearch(e: Event) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)